src/cli/initialize.js

```diff
@@ -12,37 +13,40 @@ const WELCOME_API_URL = "https://restqa.io/welcome.json";
 async function initialize(program) {
   let answers = {
     name: "app",
-    url: "https://api.restqa.io",
-    env: "local",
+    port: 8000,
     description: "Configuration generated by restqa init",
     telemetry: true
   };
 
+  const pkg = getPackageJson();
+
   if (program.y !== true) {
-    const questions = [
-      {
+    const questions = []
+
+    if (!pkg || (pkg && !pkg.name)) {
+      questions.push({
         type: "input",
         name: "name",
         default: answers.name,
         message: Locale.get("service.init.questions.name")
-      },
-      {
+      });
+    }
+
+    if (!pkg || (pkg && !pkg.description)) {
```

Comment:
Since the field `description` is optional on the package.json, what do you think of using the package name if the package description doesn't exist

Reply:
Hmm, prefer to stick to this. Feel free to let me know, why you think it's a good idea. Maybe the field description should be optional on our side too?

Reply:
It think it should be optional then..